android/assets/jsons/translations/German.properties

```diff
@@ -10,6 +10,18 @@
 StartWithCapitalLetter = true
 
 
+# Fastlane
+# These will be automatically copied to the fastlane descriptions used by F-Droid. Their keys are not as usual the english original, please read those directly as linked.
+
+# Documentation: https://f-droid.org/en/docs/Build_Metadata_Reference/#Summary
+# English to translate: https://github.com/yairm210/Unciv/blob/master/fastlane/metadata/android/en-US/short_description.txt
+Fastlane_short_description = 4X-Spiel: Zivilisationen bauen
```

Comment:
Isn't this exactly what we try to avoid with these translation files? So that translators can easily see what needs to be translated and don't have to open another link (isn't always what you want when you only have a mobile device)?

Reply:
Other than this looks fine

Reply:
Translators working from mobile? On a wearable maybe? Masochists!

Motivation: a) stands out, wake them up a little, b) full is awfully long and _much_ easier to read in the already separately existing file... c) make code independent from changes to the english text. All other translation keys are also fallbacks for immediate display (except for `" " = ` and StartWithCapitalLetter - who did those anyway?) ... so these, however long, must automatically be part of the release packages. The Fastlane english originals are _not_ displayed anywhere by Unciv itself, so I don't need the actual english content for anything other than the one offer for fastlane-speaking stores. Having the non-english text in the codebase and memory _is_ a kludge to shoehorn our translation system for these, saves some separate system, a small price to pay - that was the basic idea to this PR. Alternate concepts welcome, but puhlease don't make me escape and one-line that full_desc.

P.S.: There's still some strings in android/res/values/strings.xml _not_ using our translation central... Go take a look - abstract keys on the left.